Sources/Clickstream/Core/Interface/ClickStream.swift

```diff
@@ -52,6 +96,64 @@ public final class Clickstream {
         }
     }
     #endif
+    
+    /// Public property to get Clickstream connection state
+    public var clickstreamConnectionState: Clickstream.ConnectionState? {
+        return Clickstream.connectionState
+    }
+    
+    /// ClickStreamDataSource.
+    private weak var _dataSource: ClickStreamDataSource?
+    
+    /// readonly public accessor for dataSource.
+    public weak var dataSource: ClickStreamDataSource? {
+        get {
+            return _dataSource
+        }
+    }
+    
+    /// CSCommonProperties
+    private var _commonEventProperties: CSCommonProperties?
```

Comment:
Should we wrap this in if TRACKER_ENABLED, as CSCommonProperties is used only when health tracker is enabled?